### PR TITLE
update dotnet -i

### LINKED ISSUE #4545: Update dotnet new -i docs to document `{package-id}::{version}` syntax for installing specific template package versions
If you want to install a prerelease version of a template package you need to specify the version. By default `dotnet new` passes `*` for the version, which will only consider stable package versions. So you need to do something like `dotnet new -i Foo.Templates::0.1.0-*`.

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #4545

@danroth27 @mlorbetske please review and let me know if there's a better package for the example
I also update the version of the topic to say 2.0 since we haven't added the 2.1 options *yet*

### CODE REVIEW

**Review by danroth27 — APPROVED** [review]

**Comment by danroth27** [comment]
Looks good to me, but @mlorbetske should really review this.

**Comment by mairaw** [comment]
Build seemed stuck

File: docs/core/tools/dotnet-new.md [hunk 1]
```diff
@@ -318,6 +317,10 @@ List all templates available for MVC:
 
 `dotnet new mvc -l`
 
+Install version 2.0 of the Single Page Application templates for ASP.NET Core (command option available for .NET Core SDK 2.0 and later versions only):
```
Comment (mlorbetske) [inline docs/core/tools/dotnet-new.md]: The -i option has been available with this syntax since the 1.0 CLI. I may just be reading this wrong though.

**Review by mlorbetske — APPROVED** [review]

File: docs/core/tools/dotnet-new.md (hunk 1 above)
Comment (mairaw) [reply docs/core/tools/dotnet-new.md]: You're reading that right. Ok that makes other portions of this doc wrong then 😢 

It seems that it was introduced in 1.1 actually.

1.0.4 options:
image

1.1 options:
image

**Pull request merged**